src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/store/rca/cache/FieldDataCacheRca.java

```diff
@@ -112,21 +112,19 @@ public void setClock(Clock clock) {
         cacheEvictionCollector.collect(currTimestamp);
         if (counter >= rcaPeriod) {
             ResourceContext context;
-            HotNodeSummary nodeSummary;
-
             InstanceDetails instanceDetails = getInstanceDetails();
+            HotNodeSummary nodeSummary = new HotNodeSummary(instanceDetails.getInstanceId(), instanceDetails.getInstanceIp());
```

Comment:
So is the problem here a `null HotNodeSummary` ? Can we add UTs for a case where only few (or no) nodes have an issue?